src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -29,9 +31,32 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             dynamoViewModel = nodeView.ViewModel.DynamoViewModel;
             workspaceModel = nodeView.ViewModel.WorkspaceViewModel.Model;
 
-            var editWindowItem = new MenuItem { Header = PythonNodeModels.Properties.Resources.EditHeader, IsCheckable = false };
-            nodeView.MainContextMenu.Items.Add(editWindowItem);
-            editWindowItem.Click += delegate { EditScriptContent(); };
```

Comment:
These 3 lines were outside the `debug` flag previously. Does it not mean that they still need to be outside the `if (dynamoViewModel.IsDebugBuild)`?

Reply:
Thanks, will move out